README.md

```diff
@@ -2,16 +2,47 @@
 
 ## Introduction
 
-This is the DoS Service Fuzzy Search API.  This is a RESTFUL API which will return a list of services that match the search criteria passed to the API.  The API will perform fuzzy matching on the search criteria specified, and will fuzzy match against the following service attributes:
+This is the DoS Service Fuzzy Search API. This is a RESTFUL API which will return a list of services that match the search criteria passed to the API. The API will perform fuzzy matching on the search criteria specified, and will fuzzy match against the following service attributes:
 
 - service name
 - service public name
 - service address
 - service postcode
 
-The API endpoint is called by passing the search criteria in as query parameters.
+### Search Terms
 
-The minimum number of characters in the search criteria is set by the PARAM.VALIDATION.MIN_SEARCH_STRING_LENGTH environment variable. The maximum number of search criteria is set by the PARAM.VALIDATION.MAX_SEARCH_CRITERIA environment variable.
+The service search endpoint requires at least one search term to be passed through as a query parameter. The query parameter is as follows:
+
+    search_term
+
+The minimum number of characters allowed in a the search term is set by the CONFIGURATION.VALIDATION.MIN_SEARCH_STRING_LENGTH environment variable. The maximum number of search terms allowed is set by the CONFIGURATION.VALIDATION.MAX_SEARCH_CRITERIA environment variable.
```

Comment:
Can we wrap the variable names in the backticks to format them?

Reply:
Done